src/App.js

```diff
@@ -24,10 +24,16 @@ class App extends Component {
           </Grid>
         </Navbar>
         <Grid>
-          {/* Comment out the components to leave only the one you need to work on */}
-          <InfoPanel {...this.state} />
+          {/* Comment out the components to leave only the one you need to work on 
+
+            <InfoPanel {...this.state} />
           <NNBMap {...this.state} />
           <POIForm {...this.state} />
+          <Counter startNumber={1234} />
+
+          */}
+
+          <POIForm {...this.state} />
```

Comment:
These changes should've just been left locally - we don't want to update the master branch to have these changes. You can leave it as is while developing, but change it back to what it used to be and commit that.